DB/Item.lua

```diff
@@ -150,4 +150,19 @@ function Item:IsMount(entry)
 	return entry.type == CONSTANTS.ITEM_TYPES.MOUNT
 end
 
+function Item:HasZone(entry)
+	if not entry.coords then
+		return false
+	end
+
+	-- It has value in 'coords', make sure it contains a mapID.
+	for _, waypointData in pairs(entry.coords) do
+		if waypointData.m then
+			return true
```

Comment:
Won't this return as soon as *one* of the waypoints has a ``mapID`` assigned? If the goal of the validation is to make sure *all* waypoints care complete then this could be changed to

``if not waypointData.m then return false end``

but I'm not sure if that was the intended behaviour.

Reply:
The intention is simply to make sure the entry has a mapId set.